### DISCUSSION TITLE
[Feature] Show people with no thumbnail image into the Repair page.

### DISCUSSION THREAD

**Discussion by vincios** [immich-app/immich#9863] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

Due the issue #9840, I've a lot of person without the thumbnail image.

This happens because when the `thumbnailGeneration/generate-person-thumbnail` job fails, the person's `thumbnailPath` attribute is not being updated. But if this happen on a new person (like when the Face Detection job runs for the first time), the `thumbnailPath` attribute remains empty.

Not a big issue, since i could fix them manually (or, even better, automatically) by choosing a new featured photo. The problem is that people with empty `thumbnailPath` attribute don't show up neither in the `People` page nor in the `getAllPeople` API response.
So the only way to find these people is to query the database manually.

It would be nice if the `Repair` page would show when a person have no thumbnail image.

### Platform

- [X] Web
- [X] Server
- [ ] Mobile

**Comment by pyorot** [comment]
i would say these people could equally well be shown in the standard /people view, just with a generic silhouette thumbnail. but yeah, one or the other

i'd also consider this a bug worth filing to issues, but there are so many issues that i'm inclined to leave it here haha
